src/main/java/lotto/LottoNumbers.java

```diff
@@ -0,0 +1,133 @@
+        for (int index = 0; index < LOTTO_SIZE; index++) {
+            LottoNumber randomNumber = randomLottoNumberPocket.get(index);
+            lottoNumbers.add(randomNumber);
+        }
+    }
+
+    private void generateLottoNumbersFromString(String lottoNumbersString, String separator) {
+        lottoNumbersString = lottoNumbersString.replaceAll(FIND_ALL_SPACES, REMOVE_SPACES);
+        String[] splitLottoNumbersString = lottoNumbersString.split(separator);
+        validateLottoNumbersString(splitLottoNumbersString);
+        for (String lottoNumberString : splitLottoNumbersString) {
+            lottoNumbers.add(new LottoNumber(lottoNumberString));
+        }
+    }
+
+    private void validateLottoNumbersString(String[] splitLottoNumbersString) {
+        if (splitLottoNumbersString.length != LOTTO_SIZE) {
+            throw new WrongLottoNumberSizeException(WRONG_LOTTO_NUMBER_SIZE_MESSAGE);
+        }
+    }
+
+    public boolean containsNumber(LottoNumber myLottoNumber) {
+        return lottoNumbers.contains(myLottoNumber);
+    }
+
+    public int getMatchCount(LottoNumbers prizeLottoNumbers) {
+        return (int) lottoNumbers.stream()
+                .filter(prizeLottoNumbers::containsNumber)
+                .count();
+    }
+
+    public LottoResult getLottoResult(LottoNumbers prizeLottoNumbers) {
+        return new LottoResult(getMatchCount(prizeLottoNumbers));
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        LottoNumbers that = (LottoNumbers) o;
+        return Objects.equals(lottoNumbers, that.lottoNumbers);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(lottoNumbers);
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder result = new StringBuilder();
+        result.append(LOTTO_NUMBERS_DESCRIPTION_OPEN_BRACKET);
+        addElementForMakingString(result, lottoNumbers);
+        result.append(LOTTO_NUMBERS_DESCRIPTION_CLOSE_BRACKET);
+        return result.toString();
+    }
+
+    private void addElementForMakingString(StringBuilder result, Set<LottoNumber> lottoNumbers) {
+        List<LottoNumber> sortedLottoNumbers = convertSetToSortedLottoNumbersList(lottoNumbers);
+        int index = 0;
+        int lastIndex = sortedLottoNumbers.size() - 1;
+        for (LottoNumber lottoNumber : sortedLottoNumbers) {
+            result.append(lottoNumber.toString());
+            if (index == lastIndex) {
+                break;
+            }
+            result.append(LOTTO_NUMBERS_BASE_SEPARATOR)
+                    .append(LOTTO_NUMBERS_DESCRIPTION_SPACE);
+            index++;
+        }
+    }
+
+    private List<LottoNumber> convertSetToSortedLottoNumbersList(Set<LottoNumber> lottoNumbers) {
+        return lottoNumbers.stream()
+                .sorted(Comparator.comparingInt(LottoNumber::getNumber))
```

Comment:
```suggestion
                .sorted()
```
LottoNumber 클래스에 Comparable 인터페이스를 구현하면 간결하게 표현이 가능해요